engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -233,8 +230,11 @@ class MetadataBuilderActor(serviceRegistryActor: ActorRef) extends LoggingFSM[Me
     case Event(failure: WorkflowQueryFailure, _) =>
       context.parent ! RequestComplete(StatusCodes.BadRequest, APIResponse.fail(failure.reason))
       allDone
-    case Event(WorkflowOutputsResponse(w, metadata), _) =>
-      context.parent ! RequestComplete(StatusCodes.OK, workflowMetadataResponse(w, metadata, includeCallsIfEmpty = false))
+    case Event(WorkflowOutputsResponse(id, events), _) =>
+      // Add in an empty output event if there aren't already any output events.
+      val hasOutputs = events collectFirst { case MetadataEvent(MetadataKey(_, _, key), _, _) if key == WorkflowMetadataKeys.Outputs => () } isDefined
+      val updatedEvents = if (hasOutputs) events else MetadataEvent.empty(MetadataKey(id, None, WorkflowMetadataKeys.Outputs)) +: events
```

Comment:
I thought that the WorkflowStore was publishing an empty output event on workflow submission regardless 👍 https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/scala/cromwell/engine/workflow/WorkflowStoreActor.scala#L131